config/cesm/machines/config_compilers.xml

```diff
@@ -74,6 +74,13 @@ using a fortran linker.
   <SUPPORTS_CXX>FALSE</SUPPORTS_CXX>
 </compiler>
 
+<compiler>
```

Comment:
I think that this can go into nemo/cime_config/buildlib instead of here.

Reply:
I moved this command in nemo/cime_config/buildlib